variables.tf

```diff
@@ -98,3 +98,9 @@ variable "https_only" {
   default     = true
   type        = bool
 }
+
+variable "public_network_access_enabled" {
+  description = "(Optional) Should the Linux Web App require HTTPS connections. Defaults to `true`."
```

Comment:
question (non-blocking): Er dette den mest passende beskrivelsen for hva denne variabelen egentlig gjør?

Reply:
Du har helt rett. Her har det gått litt fort i svingene